src/pages/tasks/TaskShareDestinationSelectorModal.js

```diff
@@ -48,7 +48,6 @@ function TaskShareDestinationSelectorModal(props) {
     const [filteredPersonalDetails, setFilteredPersonalDetails] = useState([]);
     const [filteredUserToInvite, setFilteredUserToInvite] = useState(null);
 
-    // Filter out all the reports where user is not allowed to create task
```

Comment:
Not blocking on this but there was no need to remove this comment, it was actually insightful.

Reply:
Oh shoot, I must have missed that. I am sorry